src/bumper.js

```diff
@@ -20,10 +20,18 @@ import {BumperAdsController} from './bumper-ads-controller';
 import {BumperEngineDecorator} from './bumper-engine-decorator';
 import './assets/style.css';
 
+/**
+ * @enum {Object.<string, number>}}
+ */
+const BumperType: {[string]: number} = {
+  PREROLL: 0,
+  POSTROLL: -1
+};
```

Comment:
If the values represent the ad break position then shouldn't we call it BumperTypePosition or something?

Reply:
renamed to `BumperBreakType`. compare with https://github.com/kaltura/playkit-js/blob/fc25d940980eebfb9e2fb65611429e16d725ebea/src/ads/ad-break-type.js#L2

Reply:
so i guess both should be changed if they represent positions... by ok... :)